cmd/dmsg-server/commands/root.go

```diff
@@ -96,15 +101,13 @@ type Config struct {
 	LogLevel       string        `json:"log_level"`
 }
 
-func prepareMetrics(log logrus.FieldLogger, tag, addr string) servermetrics.Metrics {
+func prepareMetrics(r *chi.Mux, log logrus.FieldLogger, tag, addr string) servermetrics.Metrics {
```

Comment:
What do you think about moving the `r.Use` calls outside of the function as well? I think they should be located near router creation.

Reply:
Moved